test/test_onboard_service.sh

```diff
@@ -4,9 +4,24 @@
 source test/utils.sh
 
 KEPTN_EXAMPLES_BRANCH=${KEPTN_EXAMPLES_BRANCH:-"master"}
+KEPTN_NAMESPACE=${KEPTN_NAMESPACE:-keptn}
+KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n "$KEPTN_NAMESPACE" -o jsonpath='{.data.keptn-api-token}' | base64 --decode)
 
 echo "Testing onboarding for project $PROJECT"
 
+echo "Using remote execution plane services: ${REMOTE_EXECUTION_PLANE}"
+
+# check if REMOTE_EXECUTION_PLANE is set to true. If yes, scale down the helm-service and jmeter in the keptn namespace and install the services via their helm charts
+if [[ "${REMOTE_EXECUTION_PLANE}" == "true" ]]; then
+  kubectl scale deployment/helm-service -n "${KEPTN_NAMESPACE}" --replicas=0
+  kubectl scale deployment/jmeter-service -n "${KEPTN_NAMESPACE}" --replicas=0
+
+  KEPTN_API_HOSTNAME=$(kubectl -n "${KEPTN_NAMESPACE}" get service api-gateway-nginx -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Comment:
This should already be available in `KEPTN_ENDPOINT`, no need to fetch it here.

Reply:
yes, the `KEPTN_ENDPOINT` variable is set at this point, but with a preceding `http://` and `/api` appended, which is not  accepted by the helm chart of the jmeter/helm service

Reply:
Okay, I understand. 
I want to avoid having dependencies within the test files, in this case we depend on api-gateway-nginx being exposed via LoadBalancer. I know we have plenty of other cases where we do that, just trying to make sure we don't add new ones.

To help you with this concrete example, how about extracting `KEPTN_API_HOSTNAME` from `KEPTN_ENDPOINT`?
```
echo http://keptn.1.2.3.4.nip.io:1234/api | awk -F[/:] '{print $4}' # returns keptn.1.2.3.4.nip.io
```
Actually, we most likely would like to also get the port for the hostname, so maybe
```
echo http://keptn.1.2.3.4.nip.io:1234/api | awk -F[/] '{print $3}' # returns keptn.1.2.3.4.nip:1234
```

Reply:
sounds good, I will try out that approach